fix: disable text highlight on computer view

kt286 opened the pull request:
**before**

![image](https://user-images.githubusercontent.com/10684972/169498570-6f5d957c-e5cc-4b59-b0ff-43ab2de5e832.png)


**after**

![image](https://user-images.githubusercontent.com/10684972/169498606-062a6c70-fecc-4a02-8f52-a83f8acbfeb4.png)

max-lvs commented:
Thanks for your advice, but this definition from dde-file-mananger product design, your advice will be considered. :+1:

zccrs commented on src/dde-file-manager-lib/views/computerviewitemdelegate.cpp:
```diff
@@ -129,7 +129,7 @@ void ComputerViewItemDelegate::paint(QPainter *painter, const QStyleOptionViewIt
         painter->drawPixmap(option.rect.x() + leftmargin, option.rect.y() + topmargin, icon.pixmap(iconsize));
 
         painter->setFont(par->font());
-        painter->setPen(qApp->palette().color((option.state & QStyle::StateFlag::State_Selected) ? QPalette::ColorRole::HighlightedText : QPalette::ColorRole::Text));
```
The following talk is from the designer:

````
The text color of selected state should be use QPalette::BrightText.
````

Please try it and upload the screenshot for both dark and light theme.

kt286 replied:
![image](https://user-images.githubusercontent.com/10684972/178658528-ae195945-49c2-4fca-94e5-ea64987a40e0.png)

![image](https://user-images.githubusercontent.com/10684972/178658574-a5c166e5-8ae2-42fd-b85e-7fe0c9b0c4e3.png)

zccrs replied:
Yes, it's right.

kt286 committed:
fix: the text highlight on computer view

fix the text highlight on computer view

Log: fix the text highlight on computer view

Johnson-zs reviewed (approved):
Thanks

kt286 commented:
/merge

kt286 commented:
/merge
